src/core/components/files-regular/add.js

```diff
@@ -5,7 +5,7 @@ const all = require('async-iterator-all')
 module.exports = function (self) {
   // can't use callbackify because if `data` is a pull stream
   // it thinks we are passing a callback. This is why we can't have nice things.
-  return (data, options, callback) => {
+  return function add (data, options, callback) {
```

Comment:
Why move from arrow func to named func here?

Reply:
To have function names in any stack traces emitted making them a teensy bit easier to read.